src/pages/workspace/withPolicy.tsx

```diff
@@ -61,59 +62,51 @@ const policyPropTypes = {
     policyMembers: PropTypes.objectOf(policyMemberPropType),
 };
 
-const policyDefaultProps = {
-    policy: {},
+const policyDefaultProps: WithPolicyOnyxProps = {
```

Comment:
NAB: `WithPolicyOnyxProps` is being used before it is being declared, maybe let's move it for the sake of clarity? :)

Reply:
Changed!